contracts/interfaces/ICircuitValidator.sol

```diff
@@ -3,10 +3,7 @@ pragma solidity ^0.8.0;
 
 interface ICircuitValidator {
     struct CircuitQuery {
-        uint256 schema;
-        uint256 slotIndex;
-        uint256 operator;
-        uint256 valueHash;
+        uint256 queryHash;
```

Comment:
same here, do not remove these fields, and instead of `valueHash` you need `value` array

Reply:
fixed